docs/accessing-artifacts.adoc

```diff
@@ -1,26 +1,27 @@
 = Accessing artifacts
 
-ODS Pipeline creates and retains artifacts recording information related to a pipeline run. Those artifacts can be test reports, scan results, image digests, deployment records, etc. They are created during a pipeline run, associated with the commit being built and stored in Nexus. This document will describe how you can access the artifacts later on. A typical use case for this is attaching artifacts to verification reports for your (medical device) software.
+ODS Pipeline is able to create and retain artifacts recording information related to a pipeline run. Those artifacts may be test reports, scan results, image digests, deployment records, etc. They are created during a pipeline run, associated with the commit being built and uploaded to Nexus. This document will describe how you can access the artifacts later on. A typical use case for this is attaching artifacts to verification reports for your (regulated) software.
```

Comment:
Perhaps this is personal history only, but when reading this I assume that artifacts typically would include images as well. 
s/Those artifacts may be test reports, scan results, image digests, deployment records, etc./Those artifacts may be test reports, scan results, image digests, deployment records, etc whereas images are pushed to the cluster's image registry.

Reply:
Applied in 48e4e3b